ocamldoc/Makefile

```diff
@@ -72,16 +74,9 @@ OCAMLDOC_LIBCMI=odoc_info.cmi
 OCAMLDOC_LIBCMXA=odoc_info.cmxa
 OCAMLDOC_LIBA=odoc_info.$(A)
 
-INSTALL_LIBDIR=$(DESTDIR)$(LIBDIR)/ocamldoc
```

Comment:
This line (in the before-the-patch system) caused the bug that @sliquister just reported; if we had used the standard definition for `INSTALL_LIBDIR` , and then explicitly `$(INSTALL_LIBDIR)/ocaml` later, or a differently-name DIR variable for a non-standard definition, then we wouldn't have silently changed the behavior.